modules/rpc-util.js

```diff
@@ -0,0 +1,40 @@
+const request = require('request');
+
+class RpcRequest {
+
+    static fetchBalance(provider, accountHash) {
+        return new Promise(function(resolve, reject) {
+            var URL = `${provider}/chains/main/blocks/head/context/contracts/${accountHash}/balance`;
+
+            request(URL, function (error, response, body) {
+                if(error) {
+                    reject(error);
+                }
+                else {
+                    try {
+                        resolve(body);
+                    }
```

Comment:
in javascript, 
general formatting of try...catch is like
try {

} catch(error){

}

Reply:
why would your resolve fail. There's absolutely no need of wrapping 'resolve' call in try..catch block.  resolve() will always be successful.